### ISSUE TITLE
Material Design

### ISSUE THREAD

**Issue by aaleksandraristic** [IanTDuncan/MealTime#30]
Material 3 is the latest version of Google’s open-source design system. Design and build beautiful, usable products.

Link from the Moodle: 
https://m3.material.io/

**Comment by aaleksandraristic** [comment]
Creating buttons, UX/UI - understand the best approach to assigning this so it makes it easier for the user to use it. 

- What the user is going to use.

**Comment by BeepDroid** [comment]
Jetpack Compose is the resource for material design tools in Android Studios. I found a good well of information, along with tutorials. I originally began building the updated storyboard making a new compose project, it's pretty simple to understand, just takes a little looking. I think Jetpack would be a wonderful resource to finish the UI design, considering its capabilities.

The following links are some resources for Jetpack Compose, and I'll link the material design software I used for the updated storyboard that allowed me to package it into an APK package:

[An Android Compose Tutorial](https://developer.android.com/jetpack/compose/tutorial)

[A guide for Jetpack Compose](https://developer.android.com/jetpack/compose/documentation)

[Pathways and learning materials like assignments for Compose](https://developer.android.com/courses/jetpack-compose/course)
